ACPs/77-reinventing-subnets/README.md

```diff
@@ -505,12 +507,12 @@ def cost_over_time(V:int, T:int, x:int, Δt: int) -> int:
 
 The parameters at activation are:
 
-| Parameter | Value |
-| - | - |
-| $T$ | `TODO` Subnet Validators |
-| $C$ | `TODO` Subnet Validators |
-| $M$ | `TODO` nAVAX/s (`TODO` AVAX/day) |
-| $K$ | `TODO` |
+| Parameter | Definition                                    | Value         |
+| --------- | --------------------------------------------- | ------------- |
+| $T$       | target number of validators                   | 10_000        |
+| $C$       | capacity number of validators                 | 20_000        |
+| $M$       | minimum fee rate                              | 512 nAVAX/s   |
+| $K$       | constant to control the rate of fee changes | 1_246_488_515 |
```

Comment:
Can we put somewhere that this constant means it'll double every day?

`(10000 * 60 * 60 * 24) / (ln 2) = 1246488515.33`

Reply:
added